src/main/clojure/jarl/builtins/numbers.clj

```diff
@@ -15,8 +15,7 @@
     (set/difference a b)
     (if (and (number? a) (number? b))
       (possibly-int (-' a b))
-      (throw (errors/type-ex "minus: operand %s must be number but got %s"
-                             (if-not (number? a) 1 2) (types/->rego (if-not (number? a) a b)))))))
+      (throw (errors/type-ex "minus: operand 2 must be %s but got %s" (types/->rego a) (types/->rego b))))))
```

Comment:
If `a` is a string and `b` is an int, won't the error then say `minus: operand 2 must be string but got int`? Is that how OPA does it?

Reply:
If `a` is a string, the evaluator should not have called this function: https://github.com/open-policy-agent/opa/blob/main/capabilities.json#L2171

Reply:
Right! Then this makes sense 👍 🙂 .